BPMobileMessaging/Classes/ConactCenterCommunication/ContactCenterCommunicating.swift

```diff
@@ -4,74 +4,79 @@
 
 import Foundation
 
+/// Delegate an application should implement in order to receive events from the server.
 /// - Tag: ContactCenterEventsDelegating
 public protocol ContactCenterEventsDelegating: class {
+    /// Called when new events received from the server
+    /// - Parameter result: Array of `ContactCenterEvent` events
     func chatSessionEvents(result: Result<[ContactCenterEvent], Error>)
 }
 
 /// Provides chat and voice interactions.
 /// This API can be used for development of rich contact applications, such as customer-facing mobile and web applications for advanced chat, voice, and video communications with Bright Pattern Contact Center-based contact centers.
 /// Sends `poll` request to the backend repeatedly for get new chat events. The chat events are received through `delegate`
 public protocol ContactCenterCommunicating {
-    // MARK: - Client events delegate
-    /// Chat events delegate
+    // MARK: - Event delivery delegate
+    /// Chat events delegate.
     /// If successful returns an array of chat events [ContactCenterEvent](x-source-tag://ContactCenterEvent) for the current session that came from the server or [ContactCenterError](x-source-tag://ContactCenterError) otherwise
```

Comment:
```suggestion
    /// If successful returns an array of chat events [ContactCenterEvent](x-source-tag://ContactCenterEvent)
    /// for the current session that came from the server or [ContactCenterError](x-source-tag://ContactCenterError) otherwise
```